dev-python/cached-property/cached-property-1.5.1.ebuild

```diff
@@ -12,7 +12,7 @@ SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
 
 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="amd64 ppc64 x86"
+KEYWORDS="amd64 arm64 ppc64 x86"
```

Comment:
Don't add straight to stable, please. Please use `~am64`, initially.

Reply:
Ah, alright.